contrib/internetConnected.py

```diff
@@ -1,19 +1,10 @@
 
-import socket
-def internetConnected(host="8.8.8.8", port=53, timeout=3):
-   """
-   Simple function to quickly check, are we connected to the internet.
-   Needed to detect when waking up from sleep or returning from suspend.
-   Network can take a few seconds to reconnect. Sometimes longer.
-   Default connect to host: 8.8.8.8 (google-public-dns-a.google.com)
-   OpenPort: 53/tcp
-   Service: domain (DNS/TCP)
-   Average time to check is less than 0.2 second (200 ms).
-   """
+import requests
```

Comment:
OK.  It's a very good library, handles all the low level details.  Allows you to write very simple high level code and keep the app codebase clean.